src/java/org/apache/cassandra/index/sai/disk/v1/segment/SegmentMetadata.java

```diff
@@ -201,6 +204,19 @@ private static void writeBytes(ByteBuffer buf, IndexOutput out)
         }
     }
 
+    private static void writeBytes(byte[] bytes, IndexOutput out)
+    {
+        try
+        {
+            out.writeInt(bytes.length);
+            out.writeBytes(bytes, 0, bytes.length);
+        }
+        catch (IOException ioe)
+        {
+            throw new RuntimeException(ioe);
```

Comment:
```suggestion
            throw new UncheckedIOException(ioe);
```
Maybe in the method above too?

Reply:
I have changed this to `Throwables.unchecked`. This is what we do with these everywhere else so seems the right thing to do here.